src/actions/VaccineActions.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2021
+ */
+
+import { ToastAndroid } from 'react-native';
+import { PermissionSelectors } from '../selectors/permission';
+import selectScannedSensors from '../selectors/vaccine';
+import { PermissionActions } from './PermissionActions';
+import BleService from '../bluetooth/BleService';
+import { syncStrings } from '../localization/index';
+import { UIDatabase } from '../database/index';
+
+export const VACCINE_ACTIONS = {
+  SCAN_START: 'Vaccine/sensorScanStart',
+  SCAN_STOP: 'Vaccine/sensorScanStop',
+  SENSOR_FOUND: 'Vaccine/sensorFound',
+};
+
+const scanStart = () => ({ type: VACCINE_ACTIONS.SCAN_START });
+const scanStop = () => ({ type: VACCINE_ACTIONS.SCAN_STOP });
+const sensorFound = macAddress => ({ type: VACCINE_ACTIONS.SENSOR_FOUND, payload: { macAddress } });
```

Comment:
I prefer `id` rather than `macAddress`

Reply:
Hmm, I didn't want to get it confused with the DB field Sensor.id (which is different from Sensor.macAddress) but then this isn't really anything to do with the DB at this stage so can potentially do a rename.

Reply:
what about stuffing the whole 'device' in rather than just the id? or macAddress? is that overkill?
I just ask because I needed the `id` to make the sensor blink, but I think we'd want the `macAddress` for display purposes ( it matches the label on the back of the sensor ).
Disclaimer.. all of this is specific to Blue Maestro, so we should probs think about generic objects too!

Reply:
Annoying thing is the `id` in the `device` object returned by the scanner is actually a mac - it's a `device` object which is returned by the native module and not a `sensor` - so it is a consistent object returned (where the id is the mac, and the mac/id is used to send command via ble)

All that to say that it might be better to rename `id` below to `macAddress`?

Reply:
yup. thanks - you are spot on Mr G.

Reply:
Sounds good, should be a lot clearer now 👍  7bf78972

Reply:
ah. nice - the renaming makes it clear! thanks.